lightstep/resource_dashboard.go

```diff
@@ -28,25 +28,6 @@ func getUnifiedQuerySchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Required: true,
 		},
-		"dependency_map_options": {
```

Comment:
moved this out to `resource_metric_dashboard` since alerts dont want this in their query schema